packages/slash-command/src/types.ts

```diff
@@ -13,6 +13,7 @@ export interface TSlashInputElement extends TElement {
 
 export interface SlashRule {
   key: string;
+  text: string;
```

Comment:
```suggestion
  value: React.ReactNode;
```

To be more aligned with https://ariakit.org/reference/combobox-item#value